openownership/data-standard
# SOE technical guidance update

# Overview

- What does this pull request do?
Updates the technical guidance around SOEs to provide a scenario on modelling ownership or control by a regional state government.
- How can a reviewer test or examine your changes?
Check "Scenario 2" aligns with the related issue #447 
Check that the other svgs match their text.
- Who is best placed to review it?
@kd-ods 

(Closes/Relates to) issue: #447
